src/main/java/fakebot/Storage.java

```diff
@@ -0,0 +1,112 @@
+package fakebot;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+import fakebot.task.Task;
+import fakebot.task.TaskList;
+
+
+/**
+ * Storage class use for storing and loading data from file.
+ */
+public class Storage {
+
+    /**
+     * File name and path to be edit.
+     */
+    private String fileToEdit;
+
+    /**
+     * Constructor of storage class.
+     *
+     * @param fileName name of file.
+     * @param filePath path of file.
+     */
+    public Storage(String fileName, String filePath) {
+        filePath = createDirectory(filePath);
+        fileToEdit = filePath + fileName;
+    }
+
+    /**
+     * Creates Directory if file path doesn't exist.
+     *
+     * @param filePath path of file.
+     * @return Return path created, return empty path if path is invalid.
+     */
+    private String createDirectory(String filePath) {
+        File file = new File(filePath);
+
+        file.mkdirs();
+
+        //Remove directory and use root instead since it invalid
+        if (!file.isDirectory()) {
+            return "";
+        }
+        return filePath;
+    }
+
+    /**
+     * Writes Tasks to File.
+     *
+     * @param taskList Tasks to write to file.
+     */
+    public void writeTasksToFIle(TaskList taskList) {
```

Comment:
Perhaps you could name the method writeTasksToFile instead of writeTasksToFIle? This seems like a minor error in naming.

Reply:
will fix, thank you.